### PR TITLE
feat(dashmate): more config options for envoy and tenderdash

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

We need a more granular configuration for Envoy and Tenderdash on the testnet for stress tests

## What was done?

- Added configuration options:
  - `platform.dapi.envoy.http.connectTimeout`
  - `platform.dapi.envoy.http.responseTimeout`
  - `platform.drive.tenderdash.p2p.flushThrottleTimeout`
  - `platform.drive.tenderdash.p2p.maxPacketMsgPayloadSize`
  - `platform.drive.tenderdash.p2p.sendRate`
  - `platform.drive.tenderdash.p2p.recvRate`
  - `platform.drive.tenderdash.mempool.size`
  - `platform.drive.tenderdash.mempool.maxTxsBytes`
  - `platform.drive.tenderdash.rpc.maxOpenConnections`
  - `platform.drive.tenderdash.consensus.peer.gossipSleepDuration`
  - `platform.drive.tenderdash.consensus.peer.queryMaj23SleepDuration`
  - `platform.drive.tenderdash.consensus.unsafeOverride.propose.timeout`
  - `platform.drive.tenderdash.consensus.unsafeOverride.propose.delta`
  - `platform.drive.tenderdash.consensus.unsafeOverride.vote.timeout`
  - `platform.drive.tenderdash.consensus.unsafeOverride.vote.delta`
  - `platform.drive.tenderdash.consensus.unsafeOverride.commit.timeout`
  - `platform.drive.tenderdash.consensus.unsafeOverride.commit.bypass`

## How Has This Been Tested?

None

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/src/config/configJsonSchema.js [hunk 1]
```diff
@@ -532,23 +544,111 @@ export default {
...
+                      type: 'integer',
+                      minimum: 0,
+                    },
+                    recvRate: {
+                      type: 'integer',
+                      minimum: 0,
+                    },
                   },
-                  required: ['host', 'port', 'persistentPeers', 'seeds'],
+                  required: ['host', 'port', 'persistentPeers', 'seeds', 'flushThrottleTimeout', 'maxPacketMsgPayloadSize', 'sendRate', 'recvRate'],
```
Comment (lklimek) [inline packages/dashmate/src/config/configJsonSchema.js:563]: why are these required?

Comment (shumkov) [reply packages/dashmate/src/config/configJsonSchema.js:563]: All options in the config are required. So users can see all available options when we render the config

**Review by QuantumExplorer — APPROVED** [review]
